cve-resolution-strategy.gradle

```diff
@@ -45,15 +45,6 @@ configurations.all {
                 det.useVersion '9.4.43.v20210629'
             }
 
-            /*
```

Comment:
General note - nearly everything deleted in here was due to compatibility issues, but should now be using later versions than what it was fixed to before anyway.